docs/source/algorithms/PDFFourierTransform-v1.rst

```diff
@@ -159,26 +159,26 @@ Usage
        print('! {0:4.2f} ! {1:5f} ! {2:f} ! {3:5f} !'.format(xx[i], yy[i], Rt.readX(0)[i], Rt.readY(0)[i]))
 
 
-.. testcleanup:: ExPDFFouurierTransform
+.. testcleanup:: ExPDFFourierTransform
 
    DeleteWorkspace(ws)
    DeleteWorkspace(Rt)   
 
 **Output:**
 
-.. testoutput:: ExPDFFouurierTransform
+.. testoutput:: ExPDFFourierTransform
 
    part of S(Q) and its correlation function
-   ! 0.00 ! 1.000000 ! 0.317333 ! -3.977330 !
-   ! 0.10 ! 0.960789 ! 0.634665 ! 2.247452 !
-   ! 0.20 ! 0.852144 ! 0.951998 ! 0.449677 !
-   ! 0.30 ! 0.697676 ! 1.269330 ! 1.313403 !
-   ! 0.40 ! 0.527292 ! 1.586663 ! 0.803594 !
-   ! 0.50 ! 0.367879 ! 1.903996 ! 1.140167 !
-   ! 0.60 ! 0.236928 ! 2.221328 ! 0.900836 !
-   ! 0.70 ! 0.140858 ! 2.538661 ! 1.079278 !
-   ! 0.80 ! 0.077305 ! 2.855993 ! 0.940616 !
-   ! 0.90 ! 0.039164 ! 3.173326 ! 1.050882 !
+   ! 0.00 ! 1.000000 ! 0.317333 ! -3.977042 !
```

Comment:
What's the reason for these test results changing for v1 of the algorithm? I think for old versions the call to the algorithm made in any examples in the docs is supposed to have an extra version parameter so that the old version is called. Seems like the test results have changed here perhaps because v2 of the algorithm has been called. Have a look at the doc page for CalculateEfficiency v1 for example:

efficiency = CalculateEfficiency('workspace', MinEfficiency=0.5, MaxEfficiency=1.5, Version=1)